src/GitExtensions.SolutionRunner/UI/SolutionItemMenuItem.cs

```diff
@@ -35,11 +38,32 @@ protected override void OnClick(EventArgs e)
             process.StartInfo.UseShellExecute = true;
 
             if (settings.ShouldRunAsAdmin)
-                process.StartInfo.Verb = RunAsAdminConfiguration;
-
-            process.Start();
+                StartAdminProcess(process, e);
+            else
+                process.Start();
 
             base.OnClick(e);
         }
+
+        private void StartAdminProcess(Process process, EventArgs e)
+        {
+            if (IsVisualStudioCodeWorkspace())
+                process.StartInfo.FileName = VsCodeExe;
+
+            process.StartInfo.Verb = RunAsAdminConfiguration;
+
+            try
+            {
+                process.Start();
+            }
+            catch (Win32Exception ex)
+            {

```

Comment:
Please don't swallow the error. If user configures "runas" and don't provide path to VS, clicking on the solution file does nothing.

So ideally check executable is not set, it's not a VSCode workspace and runas is set and show some messagebox with explanation

Reply:
Thanks for the input, it was a quick fix and forgot to add here code :). I add an message box :)